circle.yml

```diff
@@ -0,0 +1,11 @@
+machine:
+  timezone:
+    America/Los_Angeles
+
+# Output the test output to circle.
+test:
+  override:
+    - go test -v ./... > test.out
```

Comment:
I noticed that the default command used by circle in e.g. https://circleci.com/gh/ClusterHQ/flocker-go/28#tests  is `go test -v -race ./...`